v-next/hardhat-node-test-coverage/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@nomicfoundation/hardhat-node-test-coverage",
+  "version": "3.0.0-next.4",
+  "description": "A node:test coverage collector for Hardhat",
+  "homepage": "https://github.com/nomicfoundation/hardhat/tree/v-next/v-next/hardhat-node-test-coverage",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NomicFoundation/hardhat",
+    "directory": "v-next/hardhat-node-test-coverage"
+  },
+  "author": "Nomic Foundation",
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": "./dist/src/collector.js"
+  },
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat"
+  ],
+  "scripts": {
+    "lint": "pnpm prettier --check && pnpm eslint",
+    "lint:fix": "pnpm prettier --write && pnpm eslint --fix",
+    "eslint": "eslint \"src/**/*.ts\" \"test/**/*.ts\"",
+    "prettier": "prettier \"**/*.{ts,js,md,json}\"",
+    "test": "node --import tsx/esm --test \"test/**/*.ts\"",
```

Comment:
We do not use our test reporter here because it would create a dependency cycle.